core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostServiceSpec.scala

```diff
@@ -23,9 +23,15 @@ class SubmissionCostServiceSpec extends FlatSpec with RawlsTestUtils {
 
   "SubmissionCostService" should "extract a map of workflow ID to cost" in {
     val expected = Map("wf1" -> 1.32f, "wf2" -> 3.00f, "wf3" -> 101.00f)
-    assertResult(expected){
+    assertResult(expected) {
       Await.result(submissionCostService.extractWorkflowCostResults(rows), 1 minute)
     }
   }
 
+  it should "bypass BigQuery with no workflow IDs" in {
+    val expected = Map.empty
+    assertResult(expected) {
+      Await.result(submissionCostService.getWorkflowCosts(Seq.empty, "test"), 1 minute)
+    }
+  }
```

Comment:
I rescind my previous comment.

Reply:
you're absolutely right! I didn't follow this all the way down. Could you please add comments explaining this, since I expect it wouldn't be obvious to other readers too?

Reply:
Done!